### ISSUE TITLE
Allow individual migrations to be ignored

### ISSUE THREAD

**Issue by argent-codes** [sbdchd/squawk#315]
Sometimes I know I am deploying an unsafe migration, but for whatever reason I've decided that this is fine. I would love the ability to flag individual migration files (maybe with a magic comment) so that Squawk doesn't report them as issues and doesn't break my CI.

**Comment by argent-codes** [comment]
This looks very similar to https://github.com/sbdchd/squawk/issues/149 actually. Not quite the same, but either would work for me, so you can close this one if you want.

**Comment by sbdchd** [comment]
https://github.com/sbdchd/squawk/issues/149 has now been fixed, so you can use:


```sql
-- squawk-ignore ban-drop-column, renaming-column,ban-drop-database
alter table t drop column c cascade;
```

But that will only ignore the rules for the following line, seems like we need a top level ignore comment
